e2e/data/campaigns-env.data.ts

```diff
@@ -0,0 +1,25 @@
+export type CampaignData = {
+  title: string
+  slug: string
+}[]
+
+const localCampaignData: CampaignData = [
+  {
+    title: 'Sunt natus qui.',
+    slug: 'sunt-natus-qui',
+  },
+  { title: 'Qui distinctio officia.', slug: 'qui-distinctio-officia' },
+]
+
+const stagingCampaignData: CampaignData = [
+  {
+    title: 'Училище за деца с нарушено зрение гр. Варна - стая за ерготерапия',
+    slug: 'uchilishe-za-deca-s-narusheno-zrenie-gr-varna-staya-za-ergoterapiya',
+  },
+  {
+    title: 'Кризисен център за пострадали от насилие - шанс за нов живот',
+    slug: 'krizisen-centur-za-postradali-ot-nasilie-shans-za-nov-zhivot',
+  },
+]
```

Comment:
Hello and thanks! 
My general assumption was that we have overall test data control, but in consequence I understood the situation. Actually I refactored all tests checking only the Dev environment (that slowed me down to realize how the project works in general). 

Regarding the tests dependencies - the best case is to expect particular data and to verify the exact values of each heading/paragraph etc., because this will be way more accurate assertion. 
Considering the situation, we the alternative is update the tests just to check the first campaign result as you suggested. 

Next couple of days I will be out of the country, so if Dimitar has the opportunity - he could make the change. 
Otherwise I will do it (maybe) next week.

Reply:
Changed in https://github.com/podkrepi-bg/frontend/commit/ee7116a07564a6b0f11c134add3a765966acc3fa